src/notification/index.js

```diff
@@ -265,29 +259,32 @@ export class NotificationListener {
   }
 }
 
-/** Try to cause a `remoteNotificationsRegistered` event. */
+/**
+ * Try to cause a `remoteNotificationsRegistered` event.
+ *
+ * Our 'register' listener will fire on that event.
+ */
 export const getNotificationToken = () => {
   if (Platform.OS === 'ios') {
-    // This leads to a call (in wix's, or RN upstream's, NotificationsIOS) to this:
+    // This leads to a call in RNC's PushNotificationIOS to this:
     //   https://developer.apple.com/documentation/uikit/uiapplication/1622932-registerusernotificationsettings
     // (deprecated after iOS 10, yikes!); which after possibly prompting the
```

Comment:
:-/ Is this still true in the current library? Dang.

Reply:
Yeah. 😦

Reply:
Er, actually, no! https://github.com/react-native-community/push-notification-ios/pull/66